libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/offboard/MapboxOffboardRouter.kt

```diff
@@ -1,278 +0,0 @@
-import com.mapbox.navigation.route.offboard.RouteBuilderProvider
-import com.mapbox.navigation.route.offboard.routerefresh.RouteRefreshCallbackMapper
-import com.mapbox.navigation.utils.internal.LoggerProvider
-import com.mapbox.navigation.utils.internal.RequestMap
-import com.mapbox.navigation.utils.internal.cancelRequest
-import okhttp3.Request
-import retrofit2.Call
-import retrofit2.Callback
-import retrofit2.Response
-
-/**
- * MapboxOffboardRouter provides online route-fetching
- *
- * @param accessToken mapboxAccessToken token
- * @param context application [Context]
- * @param urlSkuTokenProvider [UrlSkuTokenProvider]
- */
-class MapboxOffboardRouter(
-    private val accessToken: String,
-    private val context: Context,
-    private val urlSkuTokenProvider: UrlSkuTokenProvider,
-    private val logger: Logger = LoggerProvider.logger,
-) : Router {
-
-    internal companion object {
-        internal val TAG = Tag("MbxOffboardRouter")
-        private const val ROUTES_LIST_EMPTY = "routes list is empty"
-        private const val UNKNOWN = "unknown"
-    }
-
-    private val directionRequests = RequestMap<MapboxDirections>()
-    private val refreshRequests = RequestMap<MapboxDirectionsRefresh>()
-
-    /**
-     * Fetch routes based on [RouteOptions].
-     *
-     * @param routeOptions RouteOptions
-     * @param callback Callback that gets notified with the results of the request
-     *
-     * @return request ID, can be used to cancel the request with [cancelAll]
-     */
-    override fun getRoute(
-        routeOptions: RouteOptions,
-        callback: RouterCallback
-    ): Long {
-        val mapboxDirections = RouteBuilderProvider
-            .getBuilder(urlSkuTokenProvider)
-            .accessToken(accessToken)
-            .routeOptions(routeOptions)
-            .build()
-        val requestId = directionRequests.put(mapboxDirections)
-        mapboxDirections.enqueueCall(
-            object : Callback<DirectionsResponse> {
-                override fun onResponse(
-                    call: Call<DirectionsResponse>,
-                    response: Response<DirectionsResponse>
-                ) {
-                    directionRequests.remove(requestId)
-                    val urlWithoutToken = call.request().url.redactQueryParam(
-                        ACCESS_TOKEN_QUERY_PARAM
-                    ).toUrl()
-                    val body = response.body()
-                    val routes = body?.routes()
-                    val metadata = body?.metadata()?.infoMap()
-                    val code = response.code()
-                    when {
-                        call.isCanceled -> callback.onCanceled(
-                            routeOptions, RouterOrigin.Offboard
-                        )
-                        response.isSuccessful -> {
-                            logger.i(
-                                TAG,
-                                Message("Successful directions response. Metadata: $metadata"),
```

Comment:
Are these logs printed on the native side or we lost them?
They were added as part of #4760.

Reply:
cc @vanmaxim

Reply:
logging will be added on NN side